client/src/components/cityview/realm/villagers/NpcListPanel.tsx

```diff
@@ -0,0 +1,145 @@
+import { useEffect, useMemo, useState } from "react";
+import { useRoute } from "wouter";
+// import { useDojo } from "../../../../DojoContext";
+import { NpcListComponent } from "./NpcListComponent";
+import { SortPanel } from "../../../../elements/SortPanel";
+import { SortButton, SortInterface } from "../../../../elements/SortButton";
+
+export const NpcListPanel = () => {
+  // const {
+  //   setup: {
+  //     components: { Npc: NpcComponent },
+  //   //   systemCalls: { spawn_npc, change_mood },
+  //     // Not using this as the optimistic function isn't implemented
+  //   //   optimisticSystemCalls: { optimisticSpawnNpc },
+  //   },
+  //   // account: { account },
+  // } = useDojo();
```

Comment:
The useDojo will need to be used later but not the change_mood and spawn_npc functions right?

Reply:
I believe so yes, removed it for clarity.